_support.md

```diff
@@ -1,4 +1,4 @@
 # Support
-Please visit our [developer community](https://developer.uphold.com/en/community) where you can seek support from fellow developers as well as the developer support professionals at Uphold. If you need to contact us directly, please email <a href="mailto:developer@uphold.com?Subject=Help%20Me" target="_top">developer@uphold.com</a>
```

Comment:
Did this work, like ever?

Reply:
No way to know now. But we did have a community page at some point, so I assume this link was pointing correctly to that page.